domenetjenester/brevbestiller/src/main/resources/dokumentmal/informasjontilannenforelder/informasjontilannenforelder_en_GB.properties

```diff
@@ -1,4 +1,2 @@
 overskrift=Du m� s�ke om foreldrepenger i tide
-
-p1=Du kan ha rett til foreldrepenger. Du m� s�ke innen {0}, som er den siste dagen i perioden\n\
-  dere begge kan ta ut. Hvis du ikke s�ker, mister du foreldrepengene.
+du_kan_ha_rett_til_foreldrepenger=Du kan ha rett til foreldrepenger. Du m� s�ke f�r den andre forelderen har siste dag med foreldrepenger. S�ker du p� et senere tidspunkt, mister du de dagene du har s�kt for sent.
```

Comment:
sender du ikke inn dato som parameter her?

Reply:
Bruker det ikke?

Reply:
Skal ikke brukes lenger. Er visst ikke noen garanti for at fagsakene er koblet innen dette brevet trigges :)